lib/chart/internal/utils.ex

```diff
@@ -18,7 +18,14 @@ defmodule Chart.Internal.Utils do
   end
 
   def find_axis_for_vector(map, vector) do
-    map |> Enum.filter(fn {_k, ax} -> ax.vector == vector end) |> Keyword.keys()
+    {axis, _map} =
+      map
+      |> Enum.find(fn
+        {_axis, %{vector: v}} -> v == vector
+        {_k, _v} -> nil
```

Comment:
Neni lepsi vratit `false`?

Reply:
Tot otazka, ja se ridil dle [Enum.find](https://hexdocs.pm/elixir/Enum.html#find/3) specifikace:

> If no such element is found, returns `default \\ nil`.

Reply:
Specifikace je ale upne jina `find(t(), default(), (element() -> any())) :: element() | default()`. Rika to, ze prvni prvek je enumerable, druhy je nejaky default (bezne `nil`), ktery se vrati, pokud se nic nenajde a treti parametr je funkce (predikat), ktery vraci any na ktery se aplikuje `if` a pokud se vyhodnoti jako `true`, pak je prvek propusten. Neznamena to, ze pokud vratis nil, ze ten nil je ten default nebo se to s nim nejak porovnava. Ono to funguje, protoze nil ve vyrazu `if nil` se vyhodnoti jako `false`. Proto mne prijde lepsi, pokud jedna vetev fce vraci true nebo false, aby i druha zaporna vetev vracela `false`.

Reply:
Ja to jeste "cele" prepsal. Pridal jsem tam mapu `axis_table`, ktera mi drzi axis_keys a vektory a to davam do toho findu. Tedy uz zde nemusi byt ta druha vetev vracejici nil\false.